tests/fixtures/importing/create_dummy_fixtures/dummy_data_creator.py

```diff
@@ -0,0 +1,376 @@
+""" Create dummy data to populate MO """
```

Comment:
I see this is a left-over from @robertjensen's changes — although it's a useful POC we should delete if it isn't going to get used.